src/services/fileStorage/repo/files.repo.integration.test.js

```diff
@@ -0,0 +1,160 @@
+const { expect } = require('chai');
+const { FileModel } = require('../model');
+
+const {
+	findFilesThatUserCanAccess,
+	deleteFilesByIDs,
+	removeFilePermissionsByUserId,
+	findPersonalFiles,
+} = require('./files.repo');
+
+describe('user.repo.integration.test', () => {
+	let fileTestUtils;
+	let server;
+	let generateObjectId;
+	let app;
+
+	const findFiles = (fileIds = []) =>
+		FileModel.find({ _id: { $in: fileIds } })
+			.lean()
+			.exec();
+
+	before(async () => {
+		/* eslint-disable global-require */
+		app = await require('../../../app');
+		({ files: fileTestUtils, generateObjectId } = require('../../../../test/services/helpers/testObjects')(app));
+		/* eslint-enable global-require */
+		server = await app.listen(0);
+	});
+
+	after(async () => {
+		await fileTestUtils.cleanup();
+		await server.close();
+	});
+
+	describe('findPersonalFiles', () => {
+		it('no result', async () => {
+			const userToBeDeletedId = generateObjectId();
+			const fileOwnerId = generateObjectId();
+
+			await fileTestUtils.create({ owner: fileOwnerId });
+
+			const result = await findPersonalFiles(userToBeDeletedId);
+			expect(result, 'should return emptry array').to.be.an('array').with.lengthOf(0);
+		});
+
+		it('find personal files', async () => {
+			const userToBeDeletedId = generateObjectId();
+			const fileOwnerId = generateObjectId();
+
+			const ownedByUserToBeDeleted = await fileTestUtils.create({ owner: userToBeDeletedId });
+			await fileTestUtils.create({ owner: fileOwnerId });
+
+			const result = await findPersonalFiles(userToBeDeletedId);
+			expect(result, 'should return emptry array').to.be.an('array').with.lengthOf(1);
```

Comment:
I am not sure, if these kind of description bring many value. In this case maybe the default error message is enough and we don't have to deal with missleading/unupdated descriptions?

```suggestion
			expect(result, 'should return an array with one element').to.be.an('array').with.lengthOf(1);
```